GSoC-2025/movement:Calibrating confidence scores (Qianyi He).md

```diff
@@ -0,0 +1,83 @@
+## Movement: Calibrating Confidence Scores (Qianyi He)
+
+## Personal details
+- **Full name**: Qianyi He
+- **Email**: heqianyi926@uchicago.edu
+- **GitHub username**: Angelneer926
+- **Zulip username**: Qianyi He
+- **Location & time-zone**: Chicago, UTC-6
+- **Personal website / project portfolio**: Data Interaction https://github.com/Angelneer926/Data_interaction; Generative Model for Synthetic Histopathological Data Augmentation https://github.com/Angelneer926/Machine_Learning2/tree/main/Project
+- **Code contribution**
+https://github.com/neuroinformatics-unit/movement/pull/507
+
+- **Proposal discussion link**
+
+
+## **Project Proposal**
+
+### **Synopsis**  
+This project aims to calibrate confidence scores produced by pose estimation frameworks. Accurate confidence calibration ensures fair comparisons between frameworks and improves downstream tasks that rely on confidence scores, such as filtering low-confidence predictions.
+
+### **Why Is It Important?**  
+- **Fair Evaluation:** Enables objective benchmarking across different pose estimation frameworks.  
+- **Improved Performance:** Enhances confidence-based decision-making for better downstream task outcomes.  
+- **Open-Source Impact:** Provides reproducible tools and standardized evaluation practices, benefiting the open-source community.  
+
+### **Goals**  
+Implement a Python function for calibrating confidence scores produced by various pose estimation frameworks. The function will allow users to specify calibration schemes via parameters and will be accompanied by a benchmark to assess the effectiveness of the calibration methods.
```

Comment:
My impression is that the goals are better explained and captured in the section "why you"?

The contribution would likely look more like a movement module that supports a suite of techniques users can apply to calibrate confidence scores, rather than a single function.